src/uu/du/src/du.rs

```diff
@@ -720,6 +743,35 @@ Try '{} --help' for more information.",
     0
 }
 
+struct Threshold(Option<u64>, bool);
+
+impl FromStr for Threshold {
+    type Err = ParseSizeError;
+
+    fn from_str(s: &str) -> std::result::Result<Self, Self::Err> {
+        let offset = if s.starts_with('-') || s.starts_with('+') {
```

Comment:
You could combine the `starts_with` calls into `s.starts_with(&['-', '+'])`, I think. Additionally, you could cast it to a `usize`, but that is arguably less readable.

Reply:
Ah - will give it a shot.

Reply:
FYI:
there was a similar heuristic required in head/tail (and maybe others, idr):
https://github.com/uutils/coreutils/blob/440eba628c6c77875013afc4ade1b32f3f77c017/src/uu/tail/src/tail.rs#L406-L423

Reply:
Wondering if we could write another common wrapper (parse_signed_size ?) :)

Reply:
I was also considering this while writing `parse_size`, but decided against it since (at that time) I only knew about 2 files (with ~5 duplicate LOCs) that would've been applicable for this.
If you like to keep this consistent across uutils, a common wrapper would be nice. However, just copying the ~5 LOCs would also be fine.